.mailmap

```diff
@@ -0,0 +1,7 @@
+Antoine Petty <a.petty@ncl.ac.uk> <36314396+antoinepetty@users.noreply.github.com>
+Andrew Barrett-Sprot <abarrettsprot@gmail.com> <Adimote@users.noreply.github.com>
+Jake Howard <git@theorangeone.net> <RealOrangeOne@users.noreply.github.com>
+Jimmy Thompson <jimmy@jimmythompson.co.uk>
+Jimmy Thompson <jimmy@jimmythompson.co.uk> <jimmy.thompson@mendeley.com>
+Jimmy Thompson <jimmy@jimmythompson.co.uk> <jimmy.thompson@thoughtworks.com>
+Rob Spanton <rob@robspanton.com> <rob@adventurousmachines.com>
```

Comment:
Didn't Rob change his name?

Reply:
I think he did, though my aim here was to collapse the names in the history rather than to do that renaming.